cluster/cluster_test.go

```diff
@@ -7,46 +7,56 @@ import (
 )
 
 func TestPeersForQuery(t *testing.T) {
-	Init("node1", "test", time.Now())
-	ThisNode.SetPrimary(true)
-	ThisNode.SetPartitions([]int32{1, 2})
-	ThisNode.SetReady()
+	Init("node1", "test", time.Now(), "http", 6060)
+	Manager.SetPrimary(true)
+	Manager.SetPartitions([]int32{1, 2})
+	Manager.SetReady()
 	Convey("when cluster in single mode", t, func() {
 		selected := PeersForQuery()
 		So(selected, ShouldHaveLength, 1)
-		So(selected[0], ShouldEqual, ThisNode)
+		So(selected[0], ShouldResemble, Manager.ThisNode())
```

Comment:
proposal: any time we use `ShouldResemble` anywhere, we should comment about why should resemble and not equal, to make it clear to the reader. because any time i encounter this in any code base, I always wonder that same question.